README.md

```diff
@@ -1,14 +1,67 @@
-# Cosmos
+<p align="center">
 
-[![Build status](https://ci.appveyor.com/api/projects/status/kust7g5dlnykhkaf/branch/master?svg=true)](https://ci.appveyor.com/project/CosmosOS/cosmos/branch/master)
-[![Join the chat at (https://discord.com/invite/kwtBwv6jhD](https://img.shields.io/discord/833970409337913344)](https://discord.com/invite/kwtBwv6jhD)
+  <img src="https://cosmosos.github.io/images/logo.png" width="25%" />
```

Comment:
Wouldn't the full logo (#1432) work better here? (not sure myself, can't check it right now)